storage/tests/unit/test_bucket.py

```diff
@@ -54,6 +57,21 @@ def test_ctor(self):
         self.assertFalse(bucket._default_object_acl.loaded)
         self.assertIs(bucket._default_object_acl.bucket, bucket)
 
+    def test_ctor_w_user_project(self):
+        NAME = 'name'
+        USER_PROJECT = 'user-project-123'
+        connection = _Connection()
+        client = _Client(connection)
+        klass = self._get_target_class()
+        bucket = klass(client, name=NAME, user_project=USER_PROJECT)
+        self.assertEqual(bucket.name, NAME)
+        self.assertEqual(bucket._properties, {})
+        self.assertEqual(bucket.user_project, USER_PROJECT)
```

Comment:
Can you add a stand-alone test though? So that all of our unit tests are "explicit" rather than "implicit"? (i.e. each function/method has at least one test just for it)